gtk/src/Slay/Gtk.hs

```diff
@@ -369,7 +369,7 @@ exampleLayout = mkLayout $ Vis $
       substrate (LRTB 5 5 5 5) (rect $ PhaseColor $ \colorPhase -> rgb colorPhase 130 200) $
       substrate (LRTB 1 1 1 1) (rect $ PhaseConst $ rgb 0 0 0) $
       substrate (LRTB 3 3 3 3) (rect $ PhaseConst $ rgb 255 255 255) $
-      substrate (LRTB 3 3 3 3) (curve (PhaseCurvature Curvature) (PhaseColor $ \colorPhase -> rgb colorPhase 130 200) (PhaseConst (Direction True False)) ) $
+      substrate (LRTB 3 3 3 3) (curve (PhaseCurvature Curvature) (PhaseColor $ \colorPhase -> rgb colorPhase 130 200) (PhaseConst (Direction True False)) ([(2 :: Natural), (3 :: Natural), (3 :: Natural) , (5 :: Natural)]) (2 :: Natural) ) $
```

Comment:
Did you make sure an infinite list could be used here?

Reply:
They don't for now
```
src/Slay/Gtk.hs:372:168: error:
    • Could not deduce (Enum Unsigned)
        arising from the arithmetic sequence ‘2 .. ’
      from the context: HasView s El a
        bound by a type expected by the context:
                   forall s a. HasView s El a => Vis (Collage s a)
        at src/Slay/Gtk.hs:(362,17)-(380,36)
      or from: reflection-2.1.2:Data.Reflection.Reifies s1 (View El a1)
        bound by the inferred type of
                 mkMsgbox :: reflection-2.1.2:Data.Reflection.Reifies
                               s1 (View El a1) =>
                             Text -> Collage s1 a1
        at src/Slay/Gtk.hs:(368,5)-(376,72)
    • In the first argument of ‘PhaseConst’, namely
        ‘([2 .. ] :: [Unsigned])’
      In the fourth argument of ‘curve’, namely
        ‘(PhaseConst ([2 .. ] :: [Unsigned]))’
      In the second argument of ‘substrate’, namely
        ‘(curve
            (PhaseCurvature Curvature)
            (PhaseColor $ \ colorPhase -> rgb colorPhase 130 200)
            (PhaseConst (Direction True False))
            (PhaseConst ([2 .. ] :: [Unsigned]))
            (2 :: Signed))’
    
```